src/modules/Dom.js

```diff
@@ -0,0 +1,59 @@
+import ht2 from './images/heart2.png';
+// import countFoods from './countFoods.js';
+// import { addlikes, getlikes } from './likes.js';
+// import { showPopup } from './popup.js';
```

Comment:
I suggest you do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history

Reply:
This is resolved, thank you @Ambient567